src/components/NewLocationPage/ChartFooter/ChartFooter.stories.tsx

```diff
@@ -0,0 +1,153 @@
+import React, { useState } from 'react';
+import ShareButton from 'components/NewLocationPage/ShareButton/ShareButton';
+import {
+  Footer,
+  FooterSection,
+  SingleButtonWrapper,
+  ButtonContainer,
+  LegendContainer,
+  LegendContent,
+  FooterText,
+  AboutText,
+  IconWrapper,
+  ModalButton,
+} from './ChartFooter.style';
+import { useBreakpoint } from 'common/hooks';
+import ExternalLink from 'components/ExternalLink/ExternalLink';
+import NewDialog from 'components/NewDialog/NewDialog';
+
+export default {
+  title: 'Location page redesign/Chart Footer',
+};
+
+const footerText =
+  'Over the last week, Texas has averaged 3,173 new confirmed cases per day (10.9 for every 100,000 residents).';
+
+const legendIcon = (
+  <svg
+    width="8"
+    height="9"
+    viewBox="0 0 8 9"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <rect y="0.152344" width="8" height="8" fill="#E0E0E0" />
+  </svg>
+);
+
+export const ChartFooterButtonOnly = () => {
+  return (
+    <Footer>
+      <FooterSection>
+        <SingleButtonWrapper>
+          <ButtonContainer>
```

Comment:
hm do we need all these wrappers?

Reply:
Hmmm I think we will need to keep `ButtonContainer` and `SingleButtonWrapper` to prevent the button from stretching and keep the button right aligned. We can likely get rid of `FooterSection` although this would result in zero space on the right and top of the button (this is fine too if one plans to add in that space manually). `Footer` however, we can definitely get rid off in this case.